Proofreading MO Guide (#11605)
openvinotoolkit/openvino: docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_Mask_RCNN.md

> # Convert ONNX* Mask R-CNN Model {#openvino_docs_MO_DG_prepare_model_convert_model_onnx_specific_Convert_Mask_RCNN}

Author: ```suggestion
# Convert ONNX Mask R-CNN Model {#openvino_docs_MO_DG_prepare_model_convert_model_onnx_specific_Convert_Mask_RCNN}
```

> Below instructions are applicable **only** to the Mask R-CNN model converted to the ONNX* file format from the [maskrcnn-benchmark model](https://github.com/facebookresearch/maskrcnn-benchmark).

Author: ```suggestion
Below instructions are applicable **only** to the Mask R-CNN model converted to the ONNX* file format from the [maskrcnn-benchmark model](https://github.com/facebookresearch/maskrcnn-benchmark).
```
```suggestion
The instructions below are applicable **only** to the Mask R-CNN model converted to the ONNX file format from the [maskrcnn-benchmark model](https://github.com/facebookresearch/maskrcnn-benchmark).
```

> 1. Download the pre-trained model file from [onnx/models](https://github.com/onnx/models/tree/master/vision/object_detection_segmentation/mask-rcnn):

Author: ```suggestion
1. Download the pretrained model file from [onnx/models](https://github.com/onnx/models/tree/master/vision/object_detection_segmentation/mask-rcnn):
```

   * commit-SHA: 8883e49e68de7b43e263d56b9ed156dfa1e03117.

> 2. Generate the Intermediate Representation (IR) of the model, by changing your current working directory to the Model Optimizer installation directory and running the Model Optimizer with the following parameters:

Author: ```suggestion
2. Generate the Intermediate Representation (IR) of the model by changing your current working directory to the Model Optimizer installation directory and running the Model Optimizer with the following parameters:
```

> 2. Generate the Intermediate Representation (IR) of the model by changing your current working directory to the Model Optimizer installation directory and running the Model Optimizer with the following parameters:

Reviewer: ```suggestion
2. Generate the Intermediate Representation of the model by changing your current working directory to the Model Optimizer installation directory and running Model Optimizer with the following parameters:
```

--transformations_config front/onnx/mask_rcnn.json
```

> Be aware that the height and width specified with the `input_shape` command line parameter could be different. Refer to the [documentation](https://github.com/onnx/models/tree/master/vision/object_detection_segmentation/mask-rcnn) for more information about supported input image dimensions and required pre- and post-processing steps.

Author: ```suggestion
Be aware that the height and width specified with the *`input_shape`* command line parameter could be different. Refer to the [documentation](https://github.com/onnx/models/tree/master/vision/object_detection_segmentation/mask-rcnn) for more information about supported input image dimensions and required pre- and post-processing steps.
```
Do we keep "backticked" phrases in italics? Or shall we abandon the idea ?

> 3. Interpret the outputs. The generated IR file has several outputs: 

Author: ```suggestion
3. Interpret the outputs of the generated IR file: masks, class indices, probabilities and box coordinates.
```
